db/migrate/20180924185921_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,57 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[5.2]
+  def change
+
+    drop_table :users
```

Comment:
I am not sure is the best way to add missing field - dropping the table. Maybe you can only add column you need.
Add only column you will use.

Reply:
That migration was added by devise_token_auth. I'll try changing it to keep the changes without dropping the table.

Reply:
Just a comment, I told Fede to drop the table because it was faster to solve the problem of devise + devise_token_auth migration, and because there wasn't any data.  We discussed about the optimal solution to solve this (add/remove columns).